### PR TITLE
Setup demo data

### PR DESCRIPTION
**Pull request by ff-dev-45**
## Description
Created `import.sql` file that populates the database with mock data for the demo. Also refactored how the integration tests clean up data to use a single file

## How has this been tested?
Tested on intellij and all tests have passed, also ran the swagger UI and saw that the mock data was being returned through the endpoints

## Screenshots
Example showing mock course data being returned for a fake user
image

## Checklist
*(Leave blank if not applicable)*

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/main/resources/import.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+INSERT INTO COURSE (COURSE_ID, COURSE_NAME, SEMESTER, STUDENT_COUNT, TIME_UPDATED) VALUES (1, 'SOFTENG 761', 'Semester 1 2022', 9, '2022-03-16 14:29:36');
```
Comment (uoa-ypec413) [inline src/main/resources/import.sql:1]: Just wondering, do we still need the separate SQL files for each integration test?

Comment (ff-dev-45) [reply src/main/resources/import.sql:1]: currently we do still need some of them, since the data I have setup is purely just for mock data in the demo, and it doesn't add buddies or messages etc, a separate issue could be made for unifying the setup SQL files for the integration tests and then changing the tests to work for them.

Another thing is that some of the tests setup the data in a certain way to test edge cases, and this would need to be maintained when moved over. For now I think our efforts are better spent on other issues

**Review by scorpionknifes — APPROVED** [review]
"If it works it work" - famous last words

File: src/main/resources/import.sql (hunk 1 above)
Comment (uoa-ypec413) [reply src/main/resources/import.sql:1]: That makes sense, thanks for the answer :)

**Review by uoa-ypec413 — APPROVED** [review]
LGTM!
